tests/task/test_calcZernikesNeuralTask.py

```diff
@@ -0,0 +1,227 @@
+# This file is part of ts_wep.
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+# flake8: noqa
+import os
+
+from lsst.ts.wep.utils.testUtils import enforce_single_threading
+
+enforce_single_threading()
+
+import lsst.utils.tests
```

Comment:
I don't think this file is correctly isorted. Please, isntall the TSSW precommit package and do the formatting with that:

https://tssw-developer.lsst.io/development-guidelines/tools/pre_commit.html#pre-commit